README.md

```diff
@@ -1 +1,71 @@
-# TDD-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TDD project
+
+> This is a project built using test-driven development
+
+I built the folowing methods using TDD:
+
+- A method called factorial that takes one argument, an integer N, and returns the factorial for that number.
+- A method called reverse that takes one argument, a string word, and returns word reversed.
+- a method called fizzbuzz that takes one argument, an integer N, and returns a string. The returned string is constructed following these rules:
+  - When N is divisible by 3, return "fizz".
+  - When N is divisible by 5, return "buzz".
+  - When N is divisible by 3 and 5, return "fizzbuzz".
+  - Any other case, return N as a string.
+
+## Built With
+
+- Ruby
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- ruby >= 3.0.1
+
+### Install
+
+- Clone the project using 'git clone https://github.com/tiagorahal/TDD-project.git'
+
+### Run tests
+
+- Use the terminal to open the project
+- run rspecc to check tests
```

Comment:
Rspec command does not run the tests. Kindly see the screenshot below:
![test](https://user-images.githubusercontent.com/35504149/140362392-f15170ed-58f8-42ac-ac18-c5c8eae927dc.png)